nautobot/docs/media/nautobot_application_stack_low_level.txt

```diff
@@ -1 +1 @@
-https://www.canva.com/design/DAF0J8lu-9M/-X8Ke2VfMKG5ZAXKFsS_3A/edit
+https://www.canva.com/design/DAFzIltG0Ko/N4gMzo6qwylPRN0HAAfCIg/edit
```

Comment:
This link needs to be changed to view only

Reply:
Change the share permissions to "anyone with the link can view". Maybe other people should access it to see if they can edit it still.
![Screen Shot 2024-02-21 at 11 36 00 AM](https://github.com/nautobot/nautobot/assets/46973263/fe40e777-e045-4fac-bab6-b2a6c2d53eb8)

Reply:
New link appears view-only to me.